tools/spec_coverage/lib/spec.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:io';
+
+import 'package:spec_coverage/spec_parser.dart';
+
+class Spec {
+  late final List<Chapter> chapters;
+
+  Spec.fromTxt(String path) {
```

Comment:
Perhaps this could be a static method returning `Spec` (possibly renamed as `Specification`). The class could then have a private constructor accepting the list, and `chapters` wouldn't have to be late.